sendgrid/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Compassion SendGrid
```

Comment:
Not sure we should add our name in the module name (we could publish this module if generic enough)